### ISSUE TITLE
Serializing DateTime objects?

### ISSUE THREAD

**Issue by drewclauson** [nilportugues/symfony2-jsonapi-transformer#2]
How do I serialize a DateTime property? When I do so from a Doctrine entity, I get the following:

```
Notice: Undefined index: DateTime (500 Internal Server Error)
```

**Comment by nilopc** [comment]
Actually this is a bug. I just wrote the fix for it at: https://github.com/nilportugues/jsonapi-transformer/commit/16cc089912153af22450344953357067d995a1b2

Let me create a release for this fix and I'll close the issue.

**Comment by nilopc** [comment]
OK release is ready!

make sure to use `composer require nilportugues/json-api:1.0.23`

Thanks for reporting :+1: !

**Comment by drewclauson** [comment]
:+1: Thanks, I'll look into testing it tomorrow.

**Comment by drewclauson** [comment]
FYI, the name of the attribute stays camelCased instead of dasherized. 

Example from actual output:
```
"modifiedAt": {
          "date": "2011-09-07 00:00:00.000000",
          "timezone_type": 3,
          "timezone": "GMT"
        },
        "created_by_user": 0,
        "modified_by_user": 0
```

**Comment by nilopc** [comment]
hmm this needs fixing and it's easy to fix. Let me do it!

**Comment by nilopc** [comment]
@drewclauson fix included in `nilportugues/json-api:1.0.25` 

Thanks again for spotting this tiny things! :)
